src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -84,6 +84,11 @@ public class RunConfiguration : TestRunSettings
         /// </summary>
         private string targetDevice;
 
+        /// <summary>
+        /// False indicates that the test runner will use all possible adapters it can find(Nuget / Vsix / DefaultExtensions)
+        /// </summary>
+        private bool useSpecifiedAdapterLocations;
```

Comment:
Why do we still need this, I thought we decided not to have a fallback?
Can't IDE simply initialize again in case user opts in/out ?
Are we planning to let users know about this setting?